apps/meteor/server/methods/saveUserProfile.ts

```diff
@@ -128,7 +130,14 @@ async function saveUserProfile(
 				});
 			}
 
-			passwordPolicy.validate(settings.newPassword);
+			try {
```

Comment:
Shouldn't it be better to move this logic to a function to some place instead of replicating it everywhere?